Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/WEB-INF/audit.xsl

```diff
@@ -0,0 +1,394 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:n1="http://nhinc.services.com/schema/auditmessage">
+    <xsl:template match="n1:AuditMessage">
+        <html>
+            <body>
+                <!--<h3>Audit Message</h3>-->
+                <xsl:call-template name="EventIdentification"/>
+                <xsl:call-template name="ActiveParticipant"/>
+                <xsl:call-template name="AuditSourceIdentification"/>
+                <xsl:call-template name="ParticipantObjectIdentification"/>
+            </body>
+        </html>
+    </xsl:template>
+    <xsl:template name="EventIdentification">
+        <h4>Event Identification:</h4>
+        <table class="table table-bordered table-condensed">
+            <tr bgcolor="#e3ebf5">
+                <th>EventActionCode</th>
+                <th>EventDateTime</th>
+                <th>EventOutcomeIndicator</th>
+                <th>EventIDCode</th>
+                <th>EventIDdisplayName</th>
+                <th>codeSystemName</th>
```

Comment:
Do the values in this list map to something else, or should the previous two values be "EventIDDisplayName" and "CodeSystemName"?